app/models/rubygem.rb

```diff
@@ -19,7 +19,7 @@ class Rubygem < ApplicationRecord
     uniqueness: { case_sensitive: false },
     if: :needs_name_validation?
   validate :blacklist_names_exclusion
-  # validate :protected_gem_typo, on: :create, unless: -> { Array(validation_context).include?(:typo_exception) }
+  validate :protected_gem_typo, on: :create, unless: -> { Array(validation_context).include?(:typo_exception) }
```

Comment:
`"'#{name}' is too close to typo-protected gem: #{gem_typo.protected_gem}"`
we should update this. I can add my suggestion about the wording (if any) after your update.